packages/matchbox-icons/src/IconBase/IconBase.tsx

```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import PropTypes from 'prop-types';
```

Comment:
Can we uninstall this package from `matchbox-icons`? Can also remove it from the rollup build